src/modules/colorPicker/ColorPickerUI/Helpers/ColorRepresentationHelper.cs

```diff
@@ -79,9 +79,9 @@ private static string ColorToCMYK(Color color)
         /// <param name="color">The see cref="Color"/> for the hexadecimal presentation</param>
         /// <returns>A hexadecimal <see cref="string"/> representation of a RGB color</returns>
         private static string ColorToHex(Color color)
-            => $"{color.R.ToString("x2", CultureInfo.InvariantCulture)}"
-             + $"{color.G.ToString("x2", CultureInfo.InvariantCulture)}"
-             + $"{color.B.ToString("x2", CultureInfo.InvariantCulture)}";
+            => $"{color.R.ToString("X2", CultureInfo.InvariantCulture)}"
+             + $"{color.G.ToString("X2", CultureInfo.InvariantCulture)}"
+             + $"{color.B.ToString("X2", CultureInfo.InvariantCulture)}";
```

Comment:
doesn't this impact copy to clipboard format?

Reply:
I somehow mistakenly changed the Xs from lowercase to uppercase and didn't realize that. Will change them back.

Reply:
maybe for hex we should globalize that var.  We have mixmode'd this for a bit.

Reply:
This has been resolved. Please see the latest commit.